test/test_helper.rb

```diff
@@ -19,6 +19,13 @@ class ActiveSupport::TestCase
 
 class ActionController::TestCase
   include Devise::Test::ControllerHelpers
+
+  def add_resume_transcript_to(user)
+    user.resume = File.open(Rails.root.join('test/files/regular.pdf'))
+    user.transcript = File.open(Rails.root.join('test/files/regular.pdf'))
+    user.save!
+    user
+  end
```

Comment:
What are your thoughts on using FakeFS instead of including a binary in the repository?

Reply:
I'm not sure how FakeFS would fix this problem since there still needs to be a PDF to upload somewhere.